Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.11.2 as build
```

Comment:
I'm using a multi-stage docker build here. The 1st container is used to build the binaries. It does not need to be the latest version - pinning it will prevent us from surprises by sudden version upgrades. But if it's important to have it built on the latest go version, of course I can change it to "latest".

The actual (delivered) docker image is build in the second stage - it consists of the latest busybox image with the binaries created in the build stage. Nothing else from stage 1 is in this image (ensuring smallest possible image size).

Reply:
> pinning it will prevent us from surprises by sudden version upgrades

Good point.  Still, if there's a security vulnerability in the golang compiler such that it produces vulnerable binaries, I'd prefer we pick up the fix automatically.  Could we omit patch version number here?  i.e. tag it as `golang:1.11`

We bump the go minor version semi-regularly (e.g. https://github.com/containernetworking/plugins/pull/223) so we could bump this version at the same time.